index.test.js

```diff
@@ -1,5 +1,6 @@
 require("./index.js");
 ts = require("typescript");
+vueCompilerSFC = require("@vue/compiler-sfc");
```

Comment:
Can we include the script in the markup? so we would be testing our version from WG.

Reply:
I am ok with this (in module env). Now, we have html include script (in browser env) inside playground folder.